lasagne/utils.py

```diff
@@ -249,6 +249,9 @@ def create_param(spec, shape, name=None):
     support initialization with numpy arrays, existing Theano shared
     variables, and callables for generating initial parameter values.
     """
+    if any([d == 0 for d in shape]):
+        raise RuntimeError('An element in param shape is 0. shape=%r, name=%r' % (shape, name))
```

Comment:
That line is too long for pep8, please break it apart a bit. Plus, as Eben said, a `ValueError` would be more appropriate. We need to change all those `RuntimeError`s into something more suitable (#196)!
Finally, I'd slightly change the error message to: "Tried to create param %r with shape containing zeros: %r"
This hints at the fact that the creation of a parameter failed (as opposed to an existing parameter being found with a funny shape).